alacritty_terminal/src/term/mod.rs

```diff
@@ -1542,9 +1538,14 @@ impl<T: EventListener> Handler for Term<T> {
         self.goto(move_to, Column(0))
     }
 
+    /// Insert tab at cursor position.
     #[inline]
     fn put_tab(&mut self, mut count: i64) {
-        trace!("Putting tab: {}", count);
+        // A tab after the last column is the same as a linebreak
+        if self.input_needs_wrap {
+            self.wrapline();
+            return;
```

Comment:
I wasn't able to reach this code, could you share an example on how to do this?

Reply:
`echo "\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt\tt"`

Reply:
This is about tabs breaking across linewraps, so of course you need to make sure there are tabs breaking across lines.

Reply:
Oh, I see. I was doing similar thing but it wasn't working for me when I tried it first time for some reason.